src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -8406,5 +8406,49 @@ class MyAttribute : System.Attribute
 
             CompileAndVerify(source, symbolValidator: attributeValidator);
         }
+
+        [Fact, WorkItem(10639, "https://github.com/dotnet/roslyn/issues/10639")]
+        public void UsingStaticDirectiveDoesNotIgnoreObsoleteAttribute()
```

Comment:
Never mind, I see that we have similar tests for other contexts in this file.

---
In reply to: [96045994](https://github.com/dotnet/roslyn/pull/16472#discussion_r96045994) [](ancestors = 96045994)